.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,75 @@
+<!--
+Please fill in a a meaningful and concise issue title in the title field above.
+It helps us to classify the issue even  before reading the body and handling the
+issue a bit better.  Prefix the title with labels you find approciate to
+describe the issue like "[Bug]" in case of a bug, "[Feature]" for a feature
+request.
+
+Examples:
+  * Horrible: "HELP!!!"
+  * Not really okay: "Issue with fleets"
+  * Good: "[Bug] Unable to click fleet"
+  * Great: "[Bug] Unable to select fleet in galaxy map"
+-->
+
```

Comment:
This is a comment on commented out text.  If no one will see it, it probably does not need to be changed.

Prefer to list examples from best to worst.  

People are most likely to remember the first item in a list.  It may be the only item they remember.